ansible_collections/arista/cvp/examples/inventory.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
Can you remove this file from this PR? It will be added by another PR, that will be merged in before this

Reply:
fixed